appveyor.yml

```diff
@@ -20,13 +20,24 @@ skip_commits:
     - .gitignore
 
 install:
+  - ps: . .\tools\banners.ps1
+  - ps: . .\tools\build-helpers.ps1
+  - ps: SyntaxBanner
   - ps: Install-Product node '8.9.1'
-  - ps: Set-Location (Join-Path $env:APPVEYOR_BUILD_FOLDER '\tools\')
+  - ps: Set-Location $env:APPVEYOR_BUILD_FOLDER
   - ps: npm install
+  - ps: DownloadAtom
 
 build_script:
-  - ps: Set-Location (Join-Path $env:APPVEYOR_BUILD_FOLDER '\tools\')
+  - ps: BuildBanner
```

Comment:
You don't have to address this now but I want to mention that I'm hoping to try and make the build.ps1 generic enough that we can use just build.ps1 commands in the appveyor.yml.

Then we can easily drop in an appveyor.yml into any repo that follows the build.ps1 concept and we'll have CI :)